src/Main.java

```diff
@@ -0,0 +1,22 @@
+public class Main {
+    public static void main(String[] args) {
+
+        int i = 1;
+        int j = 100;
+
+        while (i <= j) {
```

Comment:
wenn du schon eine schleife hast, die eine zählervariable braucht, dann solltest du dafür am besten eine for-schleife verwenden 😉